locale/fi/LC_MESSAGES/django.po

```diff
@@ -42,6 +75,47 @@ msgstr "Pysäköintitunnusta ei löytynyt"
 msgid "Cannot change the customer of the permit"
 msgstr "Asiakasta ei voitu vaihtaa tälle pysäköintitunnukselle"
 
+msgid "You already have a valid permit for a given vehicle."
+msgstr "Sinulla on jo voimassa oleva pysäköintitunnus kyseiselle ajoneuvolle."
+
+msgid "Can not have more than 2 temporary vehicles in 365 days from first one."
+msgstr ""
+"Tilapäisten ajoneuvojen lukumäärä on rajoitettu kahteen 365 päivän aikana."
+
+msgid "Permit for a given vehicle already exist."
+msgstr "Kyseisellä ajoneuvolla on jo pysäköintitunnus."
+
+#, python-brace-format
+msgid "Customer is not an owner or holder of a vehicle {registration}"
+msgstr "Et ole ajoneuvon {registration} omistaja tai haltija"
+
+msgid "Customer does not have a valid driving licence"
+msgstr "Puutteelliset ajokorttitiedot"
+
+msgid "Non draft permit can not be deleted"
+msgstr "Pysäköintitunnusta ei voida poistaa"
+
+msgid "Contract type is required"
+msgstr "Sopimustyyppi on pakollinen"
+
+#, python-brace-format
+msgid "Only {FIXED_PERIOD} is allowed"
+msgstr "Vain  {FIXED_PERIOD} on sallittu"
+
+msgid "This is not a draft permit and can not be edited"
+msgstr "Pysäköintitunnushahmotelmaa ei voi muokata"
+
+msgid "Invalid user address."
+msgstr "Epäkelpo asiakkaan osoite"
+
+#, python-brace-format
+msgid "You can have a max of {max_allowed_permit} permits."
+msgstr "Sinulla voi olla enimmillään {max_allowed_permit} pysäköintitunnusta."
+
```

Comment:
`msgid "You can have a max of %(max_allowed_permit)d permits."`
`msgstr "Sinulla voi olla enimmillään %(max_allowed_permit)d  pysäköintitunnusta."`

Reply:
Updated